Exiled.Events/Patches/Events/Player/ItemDrop.cs

```diff
@@ -33,27 +33,29 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
         {
             List<CodeInstruction> newInstructions = ListPool<CodeInstruction>.Shared.Rent(instructions);
 
-            LocalBuilder item = generator.DeclareLocal(typeof(Item));
+            LocalBuilder item = generator.DeclareLocal(typeof(API.Features.Items.Item));
+            LocalBuilder ev = generator.DeclareLocal(typeof(DroppingItemEventArgs));
             Label returnLabel = generator.DefineLabel();
             Label notNullLabel = generator.DefineLabel();
 
             newInstructions.InsertRange(0, new[]
             {
-                // Item item = GetItem(player, itemSerial)
-                // if (item is null)
+                // if (!player.TryGetItem(itemSerial, out Item item)) {
                 //    Handlers.Player.OnDroppingNull(new DroppingNullEventArgs(Player));
```

Comment:
I think that `OnDroppingNull` looks strange, so I would like to rename that to `OnDroppingEmpty` or `OnDroppingNothing`

Reply:
> I think that `OnDroppingNull` looks strange, so I would like to rename that to `OnDroppingEmpty` or `OnDroppingNothing`

That would be a breaking change, only allowed in EXILED 6.0

Reply:
Yes, I just want to know dev bois opinion

Reply:
Personally, I don't see why OnDroppingNull even exists. I don't remember who added it or why, but I see no reason it can't still call OnDroppingItem but with a null item property.

Reply:
> Personally, I don't see why OnDroppingNull even exists. I don't remember who added it or why, but I see no reason it can't still call OnDroppingItem but with a null item property.

Okay, few reasons:

1. `ev.IsAllowed` will be useless
2. People will be catch one more NRE
3. This event will invoke when you rightclick on empty slot, a little different logic then `OnPickingUpItem`
So i dont see any reasons to delete this, but i dont like naming

Reply:
Fair points.

I think OnDroppingNothing would be the best choice for a rename.

You can implement it with the new name while keeping the original and marking it as obsolete, just make sure both events get called.

Reply:
That or comment the current event with a // TODO: so I will see it when I am prepping the 6.0 release.